pkg/rancher-desktop/components/DiagnosticsBody.vue

```diff
@@ -280,7 +281,7 @@ export default defineComponent({
         }
       }
 
-      &:not([aria-expanded]) {
+      &[aria-expanded="false"] {
```

Comment:
I think _technically_ `:not([aria-expanded="true"])` is what we want, but I don't think that matters in practice.